orchagent/bulker.h

```diff
@@ -455,6 +429,93 @@ class EntityBulker
     typename Ts::bulk_create_entry_fn                       create_entries;
     typename Ts::bulk_remove_entry_fn                       remove_entries;
     typename Ts::bulk_set_entry_attribute_fn                set_entries_attribute;
+
+    void flush_removing_entries(
+        _In_ std::vector<Te> rs)
+    {
+        size_t count = rs.size();
+        std::vector<sai_status_t> statuses(count);
+        sai_status_t status = (*remove_entries)((uint32_t)count, rs.data(), SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR, statuses.data());
+        if (status == SAI_STATUS_SUCCESS)
+        {
+            SWSS_LOG_INFO("EntityBulker.flush removing_entries %zu\n", count);
+        }
+        else
+        {
+            SWSS_LOG_ERROR("EntityBulker.flush remove entries failed, number of entries to remove: %zu, status: %s",
+                            count, sai_serialize_status(status).c_str());
+        }
+
+        for (size_t ir = 0; ir < count; ir++)
+        {
+            auto& entry = rs[ir];
+            sai_status_t *object_status = removing_entries[entry];
+            if (object_status)
+            {
+                *object_status = statuses[ir];
+            }
+        }
+    }
+
+    void flush_creating_entries(
+        _In_ std::vector<Te> rs,
```

Comment:
>vector [](http://example.com/codeflow?start=18&length=6)

Copy a vector is too heavy. Could you use reference?

Reply:
Passing reference instead.